prow/github/client.go

```diff
@@ -1271,7 +1271,8 @@ func (c *Client) GetFile(org, repo, filepath, commit string) ([]byte, error) {
 
 // Query runs a GraphQL query using shurcooL/githubql's client.
 func (c *Client) Query(ctx context.Context, q interface{}, vars map[string]interface{}) error {
-	c.log("Query", q, vars)
+	// Don't log query here because Query is typically called multiple times to get all pages.
```

Comment:
One could argue against this change and instead suggest lowering the log level if this is too noisy. I don't mind either way.

Reply:
I was considering that as well, but I think most of the other Github client calls are worth knowing about, this one is just especially noisy because it is often called multiple times per search and doesn't format very nicely. We can change this later if desired.

Reply:
The rest of github api calls are running on Info. Can we move this in Debug at least?

Reply:
Ok, if we log once per search I am fine with it.